tools/proteomiqon_psmbasedquantification/proteomiqon_psmbasedquantification.xml

```diff
@@ -0,0 +1,449 @@
+                            <when value="yes">
+                                <param name="BorderPadding" type="integer" value="2" label="Border padding amount" />
+                            </when>
+                            <when value="no">
+                            </when>
+                        </conditional>
+                        <param name="BorderPadMethod" type="select" label="Border padding method">
+                            <option value="Random" selected="true">Random</option>
+                            <option value="Zero">Zero</option>
+                        </param>
+                        <param name="InternalPaddingMethod" type="select" label="Internal padding method">
+                            <option value="Random">Random</option>
+                            <option value="NaN">NaN</option>
+                            <option value="Delete">Delete</option>
+                            <option value="Zero">Zero</option>
+                            <option value="LinearInterpolation" selected="true">Linear Interpolation</option>
+                        </param>
+                        <param name="HugeGapPaddingMethod" type="select" label="Huge gap padding method">
+                            <option value="Random">Random</option>
+                            <option value="NaN">NaN</option>
+                            <option value="Delete">Delete</option>
+                            <option value="Zero" selected="true">Zero</option>
+                            <option value="LinearInterpolation">Linear Interpolation</option>
+                        </param>
+                        <param name="HugeGapPaddingDistance" type="float" value="100.0" label="Huge gap padding distance" />
+                        <conditional name="MinPeakDistanceCond">
+                            <param name="MinPeakDistanceEnabled" type="select" label="Use min peak distance">
+                                <option value="yes">Yes</option>
+                                <option value="no" selected="true">No</option>
+                            </param> 
+                            <when value="yes">
+                                <param name="MinPeakDistance" type="float" value="2" label="Min peak distance" />
+                            </when>
+                            <when value="no">
+                            </when>
+                        </conditional>
+                        <conditional name="MinPeakLengthCond">
+                            <param name="MinPeakLengthEnabled" type="select" label="Use min peak length">
+                                <option value="yes" selected="true">Yes</option>
+                                <option value="no">No</option>
+                            </param> 
+                            <when value="yes">
+                                <param name="MinPeakLength" type="float" value="0.1" label="Min peak length" />
```

Comment:
All those conditionals with just one float you could remove and simply make the param optional if you like.

Reply:
Absolutely! Thanks :)